doc/SONiC_QoS_Mgmt_HLD-1.1.md

```diff
@@ -178,130 +178,162 @@ Here is a list of proposed new data model or existing data models from Open Conf
-  sonic(conf-sched-policy-port)# max-burst <val>
+  sonic(config)# qos scheduler-policy <name>
+  sonic(conf-sched-policy)# port
+  sonic(conf-sched-policy-port)# pir <val>
+  sonic(conf-sched-policy-port)# pbs <val>
+
+  sonic(conf-sched-policy)# no port
+
 ````
 
 - Config DSCP to Traffic Class map  
 ````
-  sonic(config)# qos-map dscp-to-tc-map <name>
+  sonic(config)# qos map dscp-tc <name>
   sonic(conf-qos-map)# dscp {<dscp-value>} traffic-class <0..7>  
   sonic(conf-qos-map)# no dscp {<dscp-value>}
 
-  sonic(config)# no dscp-to-tc-map <name>
+  sonic(config)# no qos map dscp-tc <name>
 ````
 - Config DOT1P to Traffic Class map  
 ````
-  sonic(config)# qos-map dot1p-to-tc-map <name>
+  sonic(config)# qos map dot1p-tc <name>
   sonic(conf-qos-map)# dot1p {<dot1p-value>} traffic-class <0..7>  
   sonic(conf-qos-map)# no dot1p {<dscp-value>}
 
-  sonic(config)# no dot1p-to-tc-map <name>
+  sonic(config)# no qos dot1p-tc <name>
 ````
 
 - Config Traffic Class to Queue map
 ````
-  sonic(config)# qos-map tc-to-queue-map <name>
+  sonic(config)# qos map tc-queue <name>
   sonic(conf-qos-map)# traffic-class {<tc-value>} queue <0..7>
+
+  sonic(config)# no qos map tc-queue <name>
+````
+
+- Config Traffic Class to Priority Group map
+````
+  sonic(config)# qos map tc-pg <name>
+  sonic(conf-qos-map)# traffic-class {<tc-value>} pg <0..7>
+
+  sonic(config)# no qos map tc-pg <name>
+````
+
+- Config PFC priority to queue map
 ````
+  sonic(config)# qos map pfc-priority-queue <name>
+  sonic(conf-qos-map)# pfc-priority {<priority-value>} queue <0..7>
 
+  sonic(config)# no qos map pfc-priority-queue <name>
+````
 - Config interface QoS
 ````
-  sonic(config)# qos interface <name>
-  sonic(conf-if-name)# dscp-to-tc-map <name>
-  sonic(conf-if-name)# dot1p-to-tc-map <name>
-  sonic(conf-if-name)# tc-to-queue-map <name>
-  sonic(conf-if-name)# pfc-priority <0-7>
+  sonic(config)# interface <name>
+  sonic(conf-if-name)# qos-map dscp-tc <name>
+  sonic(conf-if-name)# qos-map dot1p-tc <name>
+  sonic(conf-if-name)# qos-map tc-queue <name>
+  sonic(conf-if-name)# qos-map tc-pg <name>
+  sonic(conf-if-name)# qos-map pfc-priority-queue <name>
+  sonic(conf-if-name)# pfc priority <0-7>
   sonic(conf-if-name)# pfc asymmetric
   sonic(conf-if-name)# scheduler-policy <name>
   sonic(conf-if-name)# queue <q#> wred-profile <wred-profile-name>
-  sonic(conf-if-name)# queue <q#> [ucast|mcast] threshold <val>
 
-  sonic(conf-if-name)# no dscp-to-tc-map <name>
-  sonic(conf-if-name)# no dot1p-to-tc-map <name>
-  sonic(conf-if-name)# no tc-to-queue-map <name>
+  sonic(conf-if-name)# no qos-map dscp-tc <name>
+  sonic(conf-if-name)# no qos-map dot1p-tc <name>
+  sonic(conf-if-name)# no qos-map tc-pg <name>
+  sonic(conf-if-name)# no qos-map tc-queue <name>
+  sonic(conf-if-name)# no qos-map pfc-priority-queue <name>
   sonic(conf-if-name)# no scheduler-policy <name>
   sonic(conf-if-name)# no queue <q#> wred-profile <wred-profile-name>

```

Comment:
sonic(conf-if-name)# no queue <q#> wred-policy <wred-policy-name>

Reply:
Code is changed, missed in doc, I will correct it